src/assets/scss/components/dashboard.scss

```diff
@@ -50,6 +50,10 @@ $sectionAreaSpacing: 15px;
   }
 }
 
+.footerStyling {
```

Comment:
Since the footer is displayed on all pages, do you think it would be more appropriate to put the footer CSS in a separate file or in `main.scss`? @janakitti what are your thoughts about this?

Reply:
agreed, moved it into main.scss